packages/webview_flutter/ios/Classes/FlutterWebView.m

```diff
@@ -332,6 +340,40 @@ - (bool)loadUrl:(NSString*)url withHeaders:(NSDictionary<NSString*, NSString*>*)
   return true;
 }
 
+- (NSMutableString*)createPostParams:(NSDictionary<NSString*, NSObject*>*)params {
+  NSMutableString* resultString = [NSMutableString string];
+  for (NSString* key in [params allKeys]) {
+    if ([resultString length] > 0) [resultString appendString:@"&"];
+    [resultString appendFormat:@"%@=%@", key, [params objectForKey:key]];
```

Comment:
These values would have to be url encoded as well to match the Android implementation.
Anyway as mentioned in previous comment I think we better not make this the responsibility of the platform implementation.

Reply:
I changed implementation so developer can send encoded string by himself